pants_plugins/sendwave/pants_docker/python_requirement.py

```diff
@@ -1,21 +1,21 @@
 from dataclasses import dataclass
 
 from pants.backend.python.target_types import (PythonRequirementsField,
-                                               PythonRequirementsFileSources)
+                                               PythonRequirementsFileSourcesField)
```

Comment:
All `%Sources` target types got renamed to `%SourcesField` in 2.8.x.: https://www.pantsbuild.org/docs/plugin-upgrade-guide#28